test/test_tensorexpr.py

```diff
@@ -376,6 +376,65 @@ def easy(x, y):
         x = traced(a, b)
         np.testing.assert_allclose(a.numpy() + b.numpy(), x.numpy())
 
+    def test_double(self):
+        def easy(x, y):
+            aaa = torch.add(x, y)
+            return aaa
+
+        traced = torch.jit.trace(
+            easy,
+            (torch.rand(1024, dtype=torch.float64), torch.full((1024,), 0.5, dtype=torch.float64)),
```

Comment:
I think I set the precedent of using 1024 here but it's overkill and makes our tests kind of slow.  Let's use something small like 4.